cicd/gitlab/st/statusTracking.py

```diff
@@ -148,6 +148,7 @@ def main():
     tasks = [
     line 
     for file_name in ['submitted_tasks_TS', 'submitted_tasks_CCV', 'submitted_tasks_CV']
+    if os.path.exists(f'{work_dir}/{file_name}')
```

Comment:
missing indentation ? missing `:` at the end of line ?

Reply:
WOW... a double comprehension ? I did not know it was possible. Well... not the most "immediately clear" but I have to agree that it is correct.